src/transformers/models/wav2vec2/modeling_wav2vec2.py

```diff
@@ -44,6 +44,77 @@
 ]
 
 
+def _compute_mask_indices(
```

Comment:
This can be done in a better/faster way when coding everything in pytorch -> but training works well as it is for the moment. Will put it up as a "First Good Issue" to convert this function fully into PyTorch

Reply:
Given that it is used in the forward method and that it implies a numpy <> torch conversion, having that be all in PyTorch would probably result in a big performance boost, wouldn't it?

Ok to put it up as a good first issue!